### ISSUE TITLE
QA/QC: Masses in peak list should be checked if in scan range

### ISSUE THREAD

**Issue by tsufz** [MassBank/RMassBank#255]
As soon the scan range is known (#216), we could make use of it to enhance data quality. Peaks below or above scan range limits should no be end up in the final peak list.

**Comment by schymane** [comment]
This is _usually_ but not always valid, we have seen occasions where adducts were formed in the MS/MS and there are also cases of 2+ charged precursors (z=2) losing part of the charge and resulting in peaks above the precursor mass (m/z with z=1). Thus, care should be taken here, otherwise one might remove valuable information.

**Comment by tsufz** [comment]
Yes, I did not say that we remove them automatically, but give the user the chance to decide similar to the failed peak list. We may develop some filtering rules to check the plausibility of peaks and it could be a starting point for a more in deep search on possible adducts as a side product.
